src/screen/home.js

```diff
@@ -16,6 +17,8 @@ function App() {
   const [forecast, setForecast] = useState(null);
   const [generic, setGeneric] = useState("app");
   const [notfound, setFlag] = useState(false);
+  const [bookmarked, setBookmarked] = useState(false);
+  const [bookmarks, setBookmarks] = useState([]);
 
   console.log(1, results);
```

Comment:
does it have any purpose? if not, consider removing it

Reply:
removed unused states